build_shared.sh

```diff
@@ -65,11 +65,11 @@ for platform in $PLATFORMS; do \
                      >"$KERNEL_TMP"/build.log 2>&1;
 
                 echo "Copying new kernel image ..."
-                cp "$KERNEL_TMP/arch/arm64/boot/Image.gz-dtb" "$KERNEL_TOP/common-kernel/kernel-dtb-$device"
+                cp "$KERNEL_TMP/arch/arm64/boot/Image.gz-dtb" "$OUT_OBJ_DEST/kernel-dtb-$device"
```

Comment:
If $OUT_OBJ_DEST is set to something else, `make bootimage` will use the old kernel. Test with:
```
cd kernel/sony/msm-4.14/common-kernel
mv kernel-dtb-pdx201 kernel-dtb-pdx201.rename-to-showcase-breakage
make bootimage
...
FAILED: ninja: 'kernel/sony/msm-4.14/common-kernel/kernel-dtb-pdx201', needed by 'out/target/product/pdx201/kernel', missing and no known rule to make it
```

Could you please rework this patch only so simply that a cmdline switch copies `dtbo-${device}.img` under `$KERNEL_TMP/dtbo.img` and that would be more than sufficient:) (it would avoid being ${device} dependent later in the stack, as $KERNEL_TMP is per-device-specific.